force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -265,7 +284,7 @@ public inherited sharing class RD2_RecurringDonation {
 
             if (rd.Status__c == RD2_Constants.STATUS_PAUSED) {
                 rd.Status__c = RD2_Constants.STATUS_ACTIVE;
-                rd.ClosedReason__c = '';
+                rd.ClosedReason__c = null;
```

Comment:
This change makes it easier to compare the "old" value to the "new" value, since blank picklists are stored as `null`, and not empty strings. I don't see this causing any regression issues!